bot/utils/checks.py

```diff
@@ -67,24 +70,43 @@ def in_whitelist_check(
         # categories, it's probably not wise to rely on its category in any case.
         channels = tuple(channels) + (redirect,)
 
+    if every:
+        log.trace(
+            f"{ctx.author} may use the `{ctx.command.name}` command "
```

Comment:
why not use a triple-quoted string?

Reply:
@Kronifer If you look at the above review by Bluenix and my comment you'll see that these should in fact be one-line, so recommending usage of triple-quote string isn't correct yet.